project/src/types/offer.ts

```diff
@@ -0,0 +1,36 @@
+type Location = {
+  latitude: number,
+  longitude: number,
+  zoom: number,
+}
+
+type City = {
+	name: string,
```

Comment:
```suggestion
  name: string,
```

ну и name у города один из ШЕСТИ используй Uniontype или keyof in CityEnum